src/net/nat/mapped_tcp_socket.rs

```diff
@@ -40,7 +40,7 @@ quick_error! {
 }
 
 /// Returns a TCP socket and a set of possible external addresses of the socket. For instance, if
-/// we're behind a NAT then this will use UPnP and a pseudo-STUN protocol to try and find global IP
+/// we're behind a NAT then this will use `UPnP` and a pseudo-STUN protocol to try and find global IP
```

Comment:
hmm, why did clippy do that :/

Reply:
good question. Btw, this is the claimed issue: https://rust-lang-nursery.github.io/rust-clippy/v0.0.166/index.html#doc_markdown

Reply:
I could research this issue further and maybe even raise clippy issue?

Reply:
According to that page we should be able to add UPnP to `doc_valid_idents`.